### ISSUE TITLE
[New Framework] Explore SvelteKit

### ISSUE THREAD

**Issue by seancdavis** [seancdavis/seancdavis-com#452]
Does SvelteKit meet the requirements in #448?

**Comment by seancdavis** [comment]
SvelteKit is interesting.

First off, it's not at 1.0 yet, so I won't use it for the site. I would love to create a course for it, just to become more familiar. I really like its concepts.

SvelteKit _seems_ to be more in line with Remix (#451) in that it has strong opinions about how pages are rendered. But, from what I can tell, it will always hydrate the main loading mechanism on the client — whereas Remix works very hard to minimize the JS on the client side. (I may get in and find that SK also minimizes front end JS.)

On one hand, I really like the idea that SK is for beginners. You can get up and running with minimal JS and still build out a site. Similar to 11ty in that way. But it has mechanisms that would better scale with your site, as it doesn't have content-loading opinions.

At first look, [the way routing works](https://kit.svelte.dev/docs/routing#endpoints) feels a little weird. Having two files for the same endpoint. On the other hand, it keeps templates clean and might be a really nice way to work.

To summarize this, after some research, I'd say: **Yes, SvelteKit meets the requirements.** However, it's not at 1.0 yet, and I'm looking for a more mature framework at this point. It's between Next and Remix. But I'm going to keep my eye on SvelteKit, and hopefully use it for one of my first courses.
